documents/Documentation/Tygron percepts documentation.tex

```diff
@@ -0,0 +1,103 @@
+   & $<$SHORTNAME$>$: Short name of the stakeholder. $<$INCOME$>$: Income of the \\
+   & stakeholder. $<$START\_BUDGET$>$: Starting budget of the stakeholder.
+\end{tabular}
+
+\subsection*{Indicators}
+
+\begin{tabular}{ l l }
+  Description & Information about all the indicators of the current project.  \\
+  Type & Send once. \\
+  Syntax & indicators($<$ID$>$, $<$CUR\_VAL$>$, $<$TARG\_VAL$>$) \\
+  Parameters &  $<$ID$>$: Unique identifier for an indicator. $<$CUR\_VAL$>$: The current value of the \\
+    & indicator. $<$TARG\_VAL$>$: The target value of the indicator. 
+\end{tabular}
+
+\subsection*{Indicator link}
+
+\begin{tabular}{ l l }
+  Description & Links the stakeholders to the indicators.  \\
+  Type & Send on change. \\
+  Syntax & indicator\_link($<$NAME$>$, $<$I\_ID$>$, $<$S\_ID$>$, $<$WEIGHT$>$) \\
+  Parameters &  $<$NAME$>$: Name of an indicator for a certain stakeholder. $<$I\_ID$>$: Unique identifier \\
+    & for a indicator. $<$S\_ID$>$:  Unique identifier for a stakeholder. $<$WEIGHT$>$: The weight \\
+    & that this stakeholder has given to this indicator.
+\end{tabular}
+
+\subsection*{Zone}
+
+\begin{tabular}{ l l }
+  Description & Information about all the zones of the current project. \\
+  Type & Send on change. \\
+  Syntax & zone($<$ID$>$, $<$NAME$>$, $<$SIZE$>$, $<$MAX\_FLOORS$>$, $<$ALLOWED\_CATEGORIES$>$) \\
+  Parameters &  $<$ID$>$: Unique identifier for a zone. $<$NAME$>$: The name of the zone. \\
+    & $<$SIZE$>$: The size of the zone. $<$MAX\_FLOORS$>$: The maximum allowed floors in this \\
+    & zone. $<$ALLOWED\_CATEGORIES$>$ : The allowed categories for this zone.
+\end{tabular}
+
+\subsection*{Zone link}
+
+\begin{tabular}{ l l }
+  Description & Links the zones to the indicators.  \\
+  Type & Send on change. \\
+  Syntax & zone\_link($<$Z\_ID$>$, $<$IN\_ID$>$, $<$CUR\_VAL$>$, $<$TAR\_VAL$>$) \\
+  Parameters &  $<$Z\_ID$>$: Unique identifier for a zone. $<$IN\_ID$>$: Unique identifier for an indicator. \\
+    & $<$CUR\_VAL$>$: The current value of the indicator. $<$TAR\_VAL$>$:  The target value \\
+    & of the indicator.
+\end{tabular}
+
+\subsection*{Building}
+
+\begin{tabular}{ l l }
+  Description & Information about all the buildings of the current project.  \\
+  Type & Send on change. \\
+  Syntax & building($<$ID$>$, $<$NAME$>$, $<$CATEGORIES$>$, $<$FLOORS$>$, $<$STAKE\_ID$>$, \\
+    & $<$CONS\_YEAR$>$) \\
+  Parameters &  $<$ID$>$: Unique identifier for a building. $<$NAME$>$ : Name of the building. \\
+    & $<$CATEGORIES$>$: The categories that the building falls into. $<$FLOORS$>$:  The \\
+    & amount of floors the building has. $<$STAKE\_ID$>$: The id of the stakeholder that \\
+    & owns the building. $<$CONS\_YEAR$>$: The year the building was constructed.
+\end{tabular}
+
+\subsection*{Land}
+
+\begin{tabular}{ l l }
+  Description & Information about a piece of land.  \\
+  Type & Send on change. \\
+  Syntax & building($<$LOCATION$>$, $<$S\_ID$>$) \\
```

Comment:
Shouldn't this be land(LOCATION,ID), instead of building(LOCATION, ID)? If not, 'building' is used twice, don't know if that is a problem.